### DISCUSSION TITLE
Online Viewer Counter

### DISCUSSION THREAD

**Discussion by MinorityMeaning** [bluenviron/mediamtx#3107] [Questions]
### Question

A very powerful service. I tried it and was convinced of it. Who knows if it is possible to track the number of online viewers?

**Comment by aler9** [comment]
Hello, if your viewers are using RTSP, RTMP, WebRTC, SRT or UDP, their count can be assessed by using the Control API.

If your viewers are using HLS, this is not technically possible, see https://github.com/bluenviron/mediamtx/issues/962#issuecomment-1143654570

[a comment by an automated account is not shown]
